spec/features/local_authorities/local_authority_show_spec.rb

```diff
@@ -97,6 +97,24 @@
       expect(page).to have_link 'Edit link', href: edit_link_path(@local_authority, @service, @link.interaction)
     end
 
+    context 'editing a link' do
+      it 'returns you to the correct page after updating a link' do
+        within('.table') { click_on('Edit link', match: :first) }
+        fill_in('link_url', with: 'http://angus.example.com/link-to-change')
+        click_on('Update')
+
+        expect(page).to have_text("Services and links")
+        expect(page).to have_link('http://angus.example.com/link-to-change')
```

Comment:
Would this test pass if the page just reloaded?  What about an assertion on the URL?